dotcom-rendering/src/web/components/CaptionToggle.tsx

```diff
@@ -0,0 +1,59 @@
+import { css } from '@emotion/react';
+
+const toggleStyles = css`
+	position: absolute;
+	top: 5px;
+	right: 5px;
+	width: 32px;
+	height: 32px;
+	z-index: 1;
+	/* We're using rgba here for the opactiy */
+	background-color: rgba(18, 18, 18, 0.6);
+	border-radius: 50%;
+	border: none;
+	cursor: pointer;
+
+	svg {
+		top: 0;
+		bottom: 0;
+		right: 0;
+		left: 0;
+		margin: auto;
+		position: absolute;
+		fill: white;
+	}
+`;
+const Toggle = () => (
+	<>
+		<label htmlFor="the-checkbox" css={toggleStyles}>
+			<svg width="6" height="14" fill="white" viewBox="0 0 6 14">
```

Comment:
Note: This will be replaced by source once it has been added to the icons library. This is blocked as Akemi is currently on AL